packages/wdio-allure-reporter/src/index.js

```diff
@@ -120,6 +124,20 @@ class AllureReporter extends WDIOReporter {
         currentTest.addLabel('thread', cid)
     }
 
+    getLabels(test){
+        var tags = test.tags
+        var labels = []
+        if (tags != null && tags != undefined){
+            tags.forEach(function (tag) {
+                var label = tag.name.replace(/[@]/, '').split('=')
+                if(label.length == 2){
```

Comment:
Do you think that `length` could be more or less than `2`?

Reply:
Yes, length can be less than 2 for all other tags (other than "Allure" tags)